frontend/src/app/person/person.component.html

```diff
@@ -1,50 +1,79 @@
 <div class="row read-only-form">
   <div class="col-lg">
     <h2>Identité</h2>
-
     <div class="form-group row" *ngIf="person.mediationEnabled">
-      <label class="col-sm-6 col-form-label">Code médiation</label>
+      <label class="col-sm-6">Code médiation</label>
       <div class="col-sm-6">
         <p id="mediationCode" class="form-control-static">{{ person.mediationCode }}</p>
       </div>
     </div>
     <div class="form-group row">
-      <label class="col-sm-6 col-form-label">Est adhérent</label>
+      <label class="col-sm-6">Est adhérent</label>
       <div class="col-sm-6">
         <p id="adherent" class="form-control-static">{{ person.adherent ? 'Oui' : 'Non' }}</p>
       </div>
     </div>
     <div class="form-group row">
-      <label class="col-sm-6 col-form-label">Genre</label>
+      <label class="col-sm-6">Genre</label>
       <div class="col-sm-6">
         <p id="gender" class="form-control-static">{{ person.gender | displayGender }}</p>
       </div>
     </div>
     <div class="form-group row">
-      <label class="col-sm-6 col-form-label">Né(e) le</label>
+      <label class="col-sm-6">Né(e) le</label>
       <div class="col-sm-6">
         <p id="birthDate" class="form-control-static">{{ person.birthDate | date }}</p>
       </div>
     </div>
+    <div class="form-group row">
+      <label class="col-sm-6">Nom de naissance</label>
+      <div class="col-sm-6">
+        <p id="birthName" class="form-control-static">{{ person.birthName }}</p>
+      </div>
+    </div>
     <div class="form-group row" *ngIf="person.mediationEnabled">
-      <label class="col-sm-6 col-form-label">Statut marital</label>
+      <label class="col-sm-6">Statut marital</label>
       <div class="col-sm-6">
         <p id="maritalStatus" class="form-control-static">{{ person.maritalStatus | displayMaritalStatus }}</p>
       </div>
     </div>
     <div class="form-group row" *ngIf="person.mediationEnabled">
-      <label class="col-sm-6 col-form-label">Couverture médicale</label>
+      <label class="col-sm-6">Numéro de Sécurité Sociale</label>
+      <div class="col-sm-6">
+        <p id="socialSecurityNumber" class="form-control-static">{{ person.socialSecurityNumber }}</p>
+      </div>
+    </div>
+    <div class="form-group row" *ngIf="person.mediationEnabled">
+      <label class="col-sm-6">Numéro CAF</label>
+      <div class="col-sm-6">
+        <p id="cafNumber" class="form-control-static">{{ person.cafNumber }}</p>
+      </div>
+    </div>
+    <div class="form-group row" *ngIf="person.mediationEnabled">
+      <label class="col-sm-6">Couverture médicale</label>
       <div class="col-sm-6">
         <p id="healthCareCoverage" class="form-control-static">{{ person.healthCareCoverage | displayHealthCareCoverage }}</p>
       </div>
     </div>
+
+    <hr class="separator">
```

Comment:
Why the class separator? It doesn't do anything, apparently. Are the hr really necessary? The titles act as a separator, don't they? In particular, the last one, just before the Edit link, makes it weird because it gives the impression that the Edit link is used to edit the notes, rather than the person.

Reply:
It lets much more space between section informations (with a line).
The titles act as a separator, but the space between 2 sections is not so marked. Perhaps there is a more elegant way to increase this space?
You're right the last one, just before the Edit link, makes it weird : I delete it.